NOnion/Crypto/TorStreamCipher.fs

```diff
@@ -0,0 +1,81 @@
+﻿namespace NOnion.Crypto
+
+open System.Security.Cryptography
+open System
+
+type TorStreamCipher (keyBytes: array<byte>, ivOpt: Option<array<byte>>) =
+
+    [<Literal>]
+    let BlockSize = 16
+
+    let mutable counter: array<byte> = Array.zeroCreate BlockSize
+    let mutable counterOut: array<byte> = Array.zeroCreate BlockSize
+    let mutable keyStreamPointer: int = -1
+
+    let encryptLock: obj = obj ()
+
+    let cipher: RijndaelManaged =
```

Comment:
this variable can be created and disposed in the EncryptCounter method, it doesn't need to be stored here because it's only used there